django_instant_rest/patterns.py

```diff
@@ -5,19 +5,20 @@
 
 # Create a urlpattern element that allows CRUD
 # operations for a given model. 
-def resource(name, model, middleware = None):
+def resource(name, model, middleware = None, camel=False):
```

Comment:
@J4mesB4xter  My sloppy branch work let some changes from the camel casing PR into this one. Let's get that one taken care of first, then this one will be clearer.